k8s/upgrade/src/bin/upgrade-job/upgrade/utils.rs

```diff
@@ -29,6 +29,18 @@ pub(crate) async fn is_rebuilding(rest_client: &RestClientSet) -> Result<bool> {
         starting_token = volumes.next_token;
         for volume in volumes.entries {
             if let Some(target) = &volume.state.target {
+                let mut rebuild_count = 0;
+
+                for child in target.children.iter() {
+                    if child.rebuild_progress.is_some() {
+                        rebuild_count += 1;
+                    }
+                }
+                if rebuild_count > 0 {
+                    info!(node.name = %node_name, "Rebuilding {} of {} replicas for volume {}", rebuild_count, target.children.len(), volume.spec.uuid);
```

Comment:
Why are we adding the node-name here, these are cluster rebuilds?

Reply:
These are cluster wide rebuilds, I had added the node name just to make the logs look consistent. Removed it now.